.travis.yml

```diff
@@ -22,6 +26,12 @@ matrix:
       env: DJANGO=2.2 DB=postgres
     - python: "2.7"
       env: DJANGO=2.2 DB=sqlite
+    - python: "2.7"
+      env: DJANGO=3.0b1 DB=mysql
+    - python: "2.7"
+      env: DJANGO=3.0b1 DB=postgres
+    - python: "2.7"
```

Comment:
Also need to exclude python 3.5 with Django 3.0
https://docs.djangoproject.com/en/dev/releases/3.0/#python-compatibility